app/services/reverse_geocoding_service.rb

```diff
@@ -8,7 +8,10 @@ def initialize(lat, lon)
   end
 
   def call
-    first_result = self.class.get("/reverse", query: {lat: @lat, lon: @lon, limit: 1}).dig("features")&.first
+    response = self.class.get("/reverse", query: {lat: @lat, lon: @lon, limit: 1})
+    return nil if response.code != 200
+
+    first_result = response.dig("features")&.first
```

Comment:
De mémoire, `response` va être un objet HTTParty::Response. Cet objet accepte un reader [] tout comme les hash, mais n'implémente pas la méthode `dig`, d'où l'erreur.

En revanche, il accepte la méthode `parsed_response` qui va te retourner le corps de la réponse sous la forme d'un hash.

Tu pourrais donc faire 

```suggestion
    result = self.class.get("/reverse", query: {lat: @lat, lon: @lon, limit: 1})
    return unless result.success?
    
    response = result.parsed_response
    first_result = response.dig("features")&.first # ou response["features"]&.first
```